docs/js/alert.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Alert
+layout: page.jade
+sidebar: true
+collection: js
+priority: 3
+path: alert
+---
+
+# Alert
+<p class="lead">The alert component is an empty container where you can add a message that will be presented to users. It has an OK button by default.</p>
+
+An alert box is often used when you want to make sure information comes through to the user.
+
+### How to use
+
+You can initiate the component as a jQuery plugin:
+
+```js
+// using any selector
+$('any-selector').on('click', () => $.fn.alert(options));
+```
+
+or a vanilla constructor:
+
+```js
+import Alert from 'garden/src/js/components/alert';
+
+new Alert(options);
+```
+
+## Options
+
+The alert provides some customizable options such as: `textMessage`, `textButton`, `size`, and `triggerClose`. The default value for the `textMessage` is 'This is an example message', for the `textButton` is 'Ok', the `size` value is 'medium', and `triggerClose` is 'data-alert-button'.
+
+| Option            | Default | Description |
+|-------------------|-------------|
+| textMessage  | `This is an example message` | A text to display in the alert |
+| textButton | `Ok` | A text to the alert button |
+| size | `"medium"` | The modal size may vary between small, large, or medium |
+| triggerClose | `data-alert-button` | A string selector to bind and call the hide method when clicked |
+
+
+Ex:
+
+```js
+let options = {
+  textMessage: 'This is an example message',
+  textButton: 'Ok'
+}
+
+// as a jquery plugin
+$('[data-alert]').on('click', () => $.fn.alert(options));
+
+// as a vanilla constructor
+new Alert(options);
+```
+
+## Working with alert
+
+By default, an alert provides a text message, a text to the button, and a size value. Here's an example on how to create a button to open an alert.
+
+<div class="example example-code">
+  <button class="button button-primary" data-alert>Open Alert</button>
+</div>
+
+```js
+let alert = $('[data-alert]').on('click', () => $.fn.alert());
+```
+ Clicking on close icon or pressing the key `esc` at keyboard can hide the modal.
+
+ An alert has a default close button, but you can use `triggerClose` to delegate the `click` event to this selector and call `hide()` when the click event is fired. Example:
```

Comment:
O que acha aqui de ser mais claro sobre o `triggerClose`. Ex: `[...] you can use the triggerClose option to change the element from which the click event will close the alert`?